controller/controller.go

```diff
@@ -186,7 +200,7 @@ func (c *Controller) DeleteFunc(obj interface{}) {
 	if !finalizerskeptcontext.IsKept(ctx) {
 		c.logger.LogCtx(ctx, "event", "delete", "function", "DeleteFunc", "level", "debug", "message", "removing finalizer from runtime object")
 
-		err = c.removeFinalizer(ctx, obj)
+		err = c.removeFinalizer(obj)
```

Comment:
I did not see a reason why the context is there. It is not used for anything. It is also that for adding the finalizer there is no context taken. If we need it we can add it back again. Just want to get it straight now.